symbols/arm9.yml

```
@@ -8370,31 +8444,43 @@ arm9:
         
         r0: actor_id
         return: storage ID
-    - name: ScriptSpecialProcess0x3D
+    - name: SetActorTalkMainAndActorTalkSub
+      address:
+        NA: 0x2065B3C
+      description: |-
+        Sets ACTOR_TALK_MAIN to have actor_id.
+        Implements SPECIAL_PROC_0x3D (see ScriptSpecialProcessCall).
+        
+        r0: actor_id for ACTOR_TALK_MAIN
+        r1: actor_id for ACTOR_TALK_SUB
+    - name: SetActorTalkMain
       address:
         EU: 0x2065ECC
         NA: 0x2065B50
         JP: 0x2065E38
       description: |-
+        Sets ACTOR_TALK_MAIN to be actor_id.
         Implements SPECIAL_PROC_0x3D (see ScriptSpecialProcessCall).
         
-        No params.
-    - name: ScriptSpecialProcess0x3E
+        r0: actor_id
+    - name: SetActorTalkSub
       address:
         EU: 0x2065EDC
         NA: 0x2065B60
         JP: 0x2065E48
       description: |-
+        Sets ACTOR_TALK_SUB to be actor_id.
         Implements SPECIAL_PROC_0x3E (see ScriptSpecialProcessCall).
         
         No params.
-    - name: ScriptSpecialProcess0x17
+    - name: RandomizeDemoActors
       address:
         EU: 0x2065FC4
         NA: 0x2065C48
         JP: 0x2065F30
       description: |-
-        Implements SPECIAL_PROC_0x17 (see ScriptSpecialProcessCall).
+        Randomly picks one of the 18 teams from DEMO_TEAMS and sets ACTOR_NPC_DEMO_HERO and ACTOR_NPC_DEMO_PARTNER
```

TLDR: If someone had any footage of the kiosk demo version of EoS/EoT/EoD to review this change with, that would be nice.
I am pretty sure this was used for the Kiosk Demo versions of Explorers Of Time and Darkness as seen here: https://www.youtube.com/watch?v=naR3gy-GE7Y
However, I don't know if it's actually used in Sky because I had trouble finding footage of the Kiosk Demo version of Explorers of Sky and what footage I did see didn't seem like demo even really used the NPCs in ground mode. https://www.youtube.com/watch?v=Am5F_2TF9xY

---

I thought the demo version was the same as the final game but with the `ROM_VARIATION` scripting flag set?

---

I would probably leave off any video links unless we're absolutely certain of it. The description seems clear enough anyway without a video.

Also, is this supposed to be:

```suggestion
        Randomly picks one of the 18 teams from DEMO_TEAMS and sets ENTITY_NPC_DEMO_HERO and ENTITY_NPC_DEMO_PARTNER
```

? There's no `ACTOR_NPC` enum tags in the repo.